examples/preconditioned-solver/doc/results.dox

```diff
@@ -24,10 +25,10 @@ Solution (x):
 0.0107016
 0.0121141
 0.0123025
-Residual norm sqrt(r^T r): 
+Residual norm sqrt(r^T r):
 %%MatrixMarket matrix array real general
 1 1
-9.08137e-16
+4.82005e-08
```

Comment:
the original setting is `reduction_factor = 1e-20` and `max_iteration = 20` from https://github.com/ginkgo-project/ginkgo/blob/1cc73620651c447a8d4703b1045df41babd74d0a/examples/preconditioned-solver/preconditioned-solver.cpp. Thus, this residual changes makes sense in `reduction_factor = 1e-7`.